Makefile

```diff
@@ -22,3 +22,8 @@ bump-version:
 
 publish: build test benchmark bump-version
 	gem push lightstep-`ruby scripts/version.rb`.gem
+
+export PATH := /workspace/gopath/bin:$(PATH)
+conformance:
```

Comment:
(Should this target depend on having the conformance tested bundled by another rule? N/m I see that's difficult in Ruby--none of the other rules do this.)

I was wondering if we could shorten this snippet by just using `go run github.com/lightstep/conformance/runner.go ruby conformance_test/client.rb`, maybe. 

I take it this runs locally, assuming the `runner` binary is installed? This makes me want to rename `runner` to be more descriptive, like `conformance-runner`.

^^^ I was thinking of ways to make this Makefile work for local development, too, without requiring a docker container to run this test. The PATH above only applies to the docker container case, which makes me think there's a better way to do this. Sigh. Makefiles. Suck.